### PR TITLE
Logo Add

### PR DESCRIPTION
**Pull request by Yasujizr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<p align="center"><img src="art/Logotype/Logotype512.png" alt="Proje ismi" height="60px"></p>
```
Comment (Chan4077) [inline README.md]: Was "Proje ismi" supposed to mean something?

**Comment by Chan4077** [comment]
Just wondering, but do I have to also change the app's font? And what font is that, may I ask?

**Comment by Yasujizr** [comment]
Sorry for mistake, I'm forgetten. You can find this font on google font. Font name Baloo and you do not need to change the font of the whole program.

Font; https://fonts.google.com/?selection.family=Baloo&query=baloo

File: README.md (hunk 1 above)
Comment (Yasujizr) [reply README.md]: I'm fixed this. Project Name will be written there.

**Comment by Chan4077** [comment]
@Yasujizr BTW, do you also need me to update the web app to match the font?

**Comment by Yasujizr** [comment]
As you please, whichever is better for the app, let's do it.

**Pull request merged**